tests/runtests.pl

```diff
@@ -2779,6 +2779,7 @@ sub displaylogs {
     # a new one shortly. If all runners are busy, wait indefinitely for one to
     # finish.
     my $runnerwait = scalar(@runnersidle) && scalar(@runtests) ? 0 : undef;
+print STDERR "TESTING waiting indefinitely for test completion\n" if(!scalar(@runtests));
```

Comment:
Is this print a debugging leftover or meant to actually be a plain STDERR output? It looks like an anomaly in the code.

Reply:
This is just a debugging leftover. It won't be submitted.